packages/compose/src/higher-order/with-safe-timeout/index.tsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import { without } from 'lodash';
+// eslint-disable-next-line no-restricted-imports
+import type { ComponentType } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import createHigherOrderComponent from '../../utils/create-higher-order-component';
+// eslint-disable-next-line no-duplicate-imports
+import type { PropInjectingHigherOrderComponent } from '../../utils/create-higher-order-component';
```

Comment:
Is there a reason why we don't use only one import statement?

```suggestion
import createHigherOrderComponent, {
	PropInjectingHigherOrderComponent,
} from '../../utils/create-higher-order-component';
```

Reply:
`PropInjectingHigherOrderComponent` is a type and only ever used as a type, so it must be imported using `import type`.